src/project/render.js

```diff
@@ -0,0 +1,46 @@
+import { renderTasks } from "../task/render";
+
+let projectDivContainer = document.getElementsByClassName("projectDiv")[0];
+let rightsideTitle = document.getElementsByClassName(
+  "rightside-header-title"
+)[0];
+
+const clearProjects = () => {
+  projectDivContainer.innerText = "";
+};
+
+export const renderProjects = (projects, onProjectClick) => {
```

Comment:
So there is one thing that is important in our job. It is delegation. It's a result of isolation.

The intuition is: We will group together what belongs together. Everyone can come up with an architecture. But an architecture is tidy. If it's chaos, there is no architecture. And so it's chaotic to work with the code base. So we create folder to group things (as I did in this PR to group the project functions into different files). That's isolation.

This lead to an issue:
What if I isolate my state and my rendering from each other. How can html elements trigger state changes? We need an event listener, but as the rendering should not change the state directly, it should provide a convenient way to do so. That is delegation. 

This code here expect a `onProjectClick` function. This is `callback`. It's role is to be called when something happens (here, when a project is clicked). So this method will create the html element and can add the event listener on the click event. When it happens, call the callback. The caller of thios function will have to provide a function to handle this.